### DISCUSSION TITLE
Can't delete item

### DISCUSSION THREAD

**Discussion by giss69** [dani-garcia/vaultwarden#5263] [Q&A] [answered]
Hello,

I can't delete items anymore. actually  I didn't change anything. If I trying to delete items vaulwarden logged me out. I tried it via web interface, safari app, Mac app, iPhone app., same behaviour (log out).
Vaultwarden running in docker is actually 1.32.5. all apps are up to day

what went wrong with system? I'm using it since 4 years without issues.

thank you

**Comment by BlackDex** [comment]
Provide some logs of both your reverse proxy and Vaultwarden.

**Comment by giss69** [comment]
@BlackDex 
do you mean docker logs?

**Reply by BlackDex** [reply]
If you are using docker, then yes.

**Comment by giss69** [comment, marked as answer]
I guess, I found the problem. It was a reverse proxy issue not vaultwarden.

**Reply by 0xFFA500** [reply]
I'm experiencing this issue, please may you let me know the reverse proxy you were using and what you did to fix the problem?

**Reply by giss69** [reply]
Hi,
I'm using zoraxy reverse proxy. I just did a fresh voultwarden setup on proxy manager and what fixed my problem.

**Reply by shodeira** [reply]
> I'm experiencing this issue, please may you let me know the reverse proxy you were using and what you did to fix the problem?

You maybe have this issue: https://github.com/dani-garcia/vaultwarden/discussions/5996

**Reply by 0xFFA500** [reply]
> > I'm experiencing this issue, please may you let me know the reverse proxy you were using and what you did to fix the problem?
> 
> You maybe have this issue: #5996

I'm also having the problem in all apps including desktop and the Firefox, the only place I can delete items is the web
I have quite a few attachments, so unfortunately I can't easily export and import to see if a reinstall will help

**Reply by stefan0xC** [reply]
@oscarorange the fix #6004 is currently only available in `testing` so either use that or downgrade your clients to a version before `v2025.6.0`.

**Reply by 0xFFA500** [reply]
> @oscarorange the fix #6004 is currently only available in `testing` so either use that or downgrade your clients to a version before `v2025.6.0`.

Thanks!
